src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -146,7 +151,10 @@ export default function ConfirmedAppointmentListItem({
         className="vads-u-font-size--sm vads-u-font-weight--normal vads-u-font-family--sans"
       >
         <span className="vaos-form__title">{header}</span>
-        <span>{vvcHeader}</span>
+        <span>
+          {vvcHeader}
+          {phoneApptHeader}
```

Comment:
Instead of adding a new `phoneApptHeader` variable, could we rename `vvcHeader` and use that for both vvc and phone appts? There's only one ever one of these header tags, plus if you keep the two variables, you also have to modify the `AddToCalendar` component below to use `vvcHeader` or `phoneApptHeader`.

Reply:
changed `vvcHeader` to `subHeader` to be generic enough to include sub text for either video or phone appts.